parm/config/gefs/config.base.emc.dyn

```diff
@@ -220,9 +220,10 @@ export gfs_cyc=@gfs_cyc@ # 0: no GFS cycle, 1: 00Z only, 2: 00Z and 12Z only, 4:
 export FHMIN_GFS=0
 export FHMIN=${FHMIN_GFS}
 export FHMAX_GFS=@FHMAX_GFS@
-export FHOUT_GFS=6 # Must be 6 for S2S until #1629 is addressed; 3 for ops
+export FHOUT_GFS=6 
```

Comment:
Thank you for also making parallel updates in the GEFS configs.

Reply:
I think there is still more work that's needed in workflow/rocoto/gefs_tasks.py but was a bit at a loss as to what/where this needs to be added since I didn't see ocean/ice output there

Reply:
Opened a quick (untested) PR into this branch

Reply:
GEFS should be handled now with this change in.